YTIBrowseEndpoint.h

```diff
@@ -0,0 +1,4 @@
+#import <Foundation/Foundation.h>
+@interface YTIBrowseEndpoint : NSObject
+@property(copy, nonatomic) NSString *browseId;
```

Comment:
```suggestion
@property (copy, nonatomic) NSString *browseId;
```
For every file that has one or more properties, check this spacing.